ticdc: update ticdc time-zone docs (#13850)
pingcap/docs: ticdc/troubleshoot-ticdc.md

Warning: Unable to load '/usr/share/zoneinfo/zone1970.tab' as time zone. Skipping it.
```

> If the downstream is a special MySQL environment (a public cloud RDS or some MySQL derivative versions) and importing the time zone using the above method fails, can use the downstream default time zone by setting `time-zone` to a empty value, e.g. `time-zone=""`.

Reviewer: ```suggestion
If the downstream is a special MySQL environment (a public cloud RDS or some MySQL derivative versions) and importing the time zone using the preceding method fails, you can use the default time zone of the downstream by setting `time-zone` to an empty value, such as `time-zone=""`.
```

> When using time zones in TiCDC, it is recommended to explicitly specify the time zone, such as `time-zone="Asia/Shanghai"`. Also, make sure that the `tz`, and the time zone configuration of the downstream database are consistent. This prevents the data inconsistency caused by inconsistent time zones.

Reviewer 2: ```suggestion
When using time zones in TiCDC, it is recommended to explicitly specify the time zone, such as `time-zone="Asia/Shanghai"`. Also, make sure that `tz` specified in TiCDC server configurations and `time-zone` specified in Sink URI are consistent with the time zone configuration of the downstream database. This prevents data inconsistency caused by inconsistent time zones.
```
